README.md

```diff
@@ -49,6 +49,12 @@ For example, if you need facebook login, include the following
 pod 'SKYKit/Facebook'
 ```
 
+For forgot password integration, you can include the following
+
+```
+pod 'SKYKit/ForgotPassword'
```

Comment:
I think the the `SKYKit/ForgotPassword` subspec should be included in the `SKYKit` spec by default. The README here should be updated to reflect this.